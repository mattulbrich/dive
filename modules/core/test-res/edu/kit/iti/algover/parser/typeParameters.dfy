
// to be extended later with parametrised classes, parametrised types etc

method m() 
{
   $internal<Param1, Param2 < Param3, Param4 >>($internal2<X>, 0);
}
