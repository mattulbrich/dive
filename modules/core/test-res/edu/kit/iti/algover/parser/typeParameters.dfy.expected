(COMPILATION_UNIT 
 (method m ARGS 
  (BLOCK 
   (CALL 
    ($internal Param1 (Param2 Param3 Param4)) 
    (ARGS ($internal2 X) 0)))))
